tests/ThreeWayMergeTest.php

```diff
@@ -196,12 +196,783 @@ public function testConflict()
         ];
 
         $merge = new ThreeWayMerge();
-        $merge->performMerge($original, $local, $remote);
         try {
             $merge->performMerge($original, $local, $remote);
             $this->fail('Exception was not thrown.');
-        } catch (Exception $e) {
+        } catch (ConflictException $e) {
             $this->assertTrue(true);
         }
     }
+
+    /**
+     * Test for multiline modification
+     * in same number of lines.
+     *
+     * @throws ConflictException
```

Comment:
None of the test methods throws `ConflictException`. In the cases we do test conflicts we catch the exception, so it's not thrown in that sense. So let's remove this line for all test methods.